### DISCUSSION TITLE
disable Flashlight or similar mod in replay like stable

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#18188] [Ideas] [answered]
In stable, when I watch a replay that play with FL. I can close FL (or make the hidden part translucent) by clicking FL mod icon
It also worked with DT in stable

but it not implemented on lazer, is it not intended to provide this functionality or is it just not done?

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/8357.
